topsort-api-v2.yml

```diff
@@ -379,17 +375,20 @@ components:
             $ref: '#/components/schemas/AssetSource'
           minLength: 1
 
-    Product:
+    Products:
       required:
-        - id
+        - ids
       type: object
       properties:
-        id:
-          type: string
-          description: >
-            The marketplace's ID of the product which will participate in the auction.
-            These ID must match those in the catalog integration with Topsort.
-          example: p_SA0238
+        ids:
```

Comment:
@roalonso Should we add `minItems`?

Reply:
Ahh yes, and `maxItems`. Or should we add that limit later? we never discussed how many product ID we want to receive at most per request.

Reply:
lets say 10k, that's higher than what we are receiving now